waku/v2/node/wakunode2.nim

```diff
@@ -701,6 +703,78 @@ proc connectToNodes*(n: WakuNode, nodes: seq[RemotePeerInfo]) {.async.} =
   # later.
   await sleepAsync(5.seconds)
 
+proc runDiscv5Loop(node: WakuNode) {.async.} =
```

Comment:
Maybe this could return a closure/channel whereby you can stop the discovery loop? Could be useful for clients. Or do you expect this behavior to be done with start/stop manually?

EDIT: Nevermind, start calls this and it is already possible